politeiawww/userwww.go

```diff
@@ -92,35 +142,56 @@ func (p *politeiawww) getSessionUser(w http.ResponseWriter, r *http.Request) (*u
 	return user, nil
 }
 
-// setSessionUserID sets the "uuid" session key to the provided value.
-func (p *politeiawww) setSessionUserID(w http.ResponseWriter, r *http.Request, id string) error {
-	log.Tracef("setSessionUserID: %v %v", id, www.CookieSession)
-	session, err := p.getSession(r)
+// initSession adds a session record to the db and sets its ID in the cookie.
+func (p *politeiawww) initSession(w http.ResponseWriter, r *http.Request, uid string) error {
+	log.Tracef("initSession: %v %v", uid, www.CookieSession)
+	cookie, err := p.getCookie(r)
 	if err != nil {
 		return err
 	}
 
-	session.Values["uuid"] = id
-	return session.Save(r, w)
+	pid, err := uuid.Parse(uid)
+	if err != nil {
+		return err
+	}
+
+	sessionid := uuid.New()
+	err = p.db.SessionNew(user.Session{
+		ID:     sessionid,
+		UserID: pid,
+		MaxAge: int64(p.store.Options.MaxAge),
```

Comment:
I think this should be using `sessionMaxAge`, right?

https://github.com/decred/politeia/blob/e8c24897e2dfbf256e1f7cfeb56a7184ec54997b/politeiawww/www.go#L52

Reply:
Right. Fixed.